bluetooth/bluetooth_driver.c

```diff
@@ -16,46 +16,59 @@
 
 #include <stdint.h>
 
-#ifdef HAVE_CONFIG_H
-#include "../config.h"
+#ifdef HAVE_CONFIG_H#include "../config.h"
+
 #endif
 
 #include "../driver.h"
+
 #include "../list_special.h"
+
 #include "../retroarch.h"
+
 #include "../runloop.h"
+
 #include "../verbosity.h"
 
 #include "bluetooth_driver.h"
 
 static bluetooth_driver_t bluetooth_null = {
-   NULL, /* init */
-   NULL, /* free */
-   NULL, /* scan */
-   NULL, /* get_devices */
-   NULL, /* device_is_connected */
-   NULL, /* device_get_sublabel */
-   NULL, /* connect_device */
-   NULL, /* remove_device */
-   "null",
+  NULL,
+  /* init */
+  NULL,
+  /* free */
+  NULL,
+  /* scan */
+  NULL,
+  /* get_devices */
+  NULL,
+  /* device_is_connected */
+  NULL,
+  /* device_get_sublabel */
+  NULL,
+  /* connect_device */
+  NULL,
+  /* remove_device */
+  "null",
 };
 
-const bluetooth_driver_t *bluetooth_drivers[] = {
-#ifdef HAVE_BLUETOOTH
-   &bluetooth_bluetoothctl,
-#ifdef HAVE_DBUS
-   &bluetooth_bluez,
-#endif
-#endif
-   &bluetooth_null,
-   NULL,
+const bluetooth_driver_t * bluetooth_drivers[] = {
+  #ifdef HAVE_BLUETOOTH &
+  bluetooth_bluetoothctl,
+  #ifdef HAVE_DBUS &
+  bluetooth_bluez,

```

Comment:
Having the `&` on the same line as the macro is confusing. Goes against the majority of the other RetroArch code standards.